packages/webcomponents/stencil.config.ts

```diff
@@ -18,6 +22,19 @@ export const config: Config = {
       file: './dist/docs.json',
     },
   ],
+  plugins: [
+    replace({
+      PROXY_API_ORIGIN: JSON.stringify(process.env.PROXY_API_ORIGIN),
+      IFRAME_ORIGIN: JSON.stringify(process.env.IFRAME_ORIGIN),
+    }),
+  ],
   validatePrimaryPackageOutputTarget: true,
   sourceMap: false,
+  testing: {
+    setupFiles: ['./setupTests.js'],
+  },
+  buildEs5: false,
+  // Setting the minify flags to false will help retain console.logs
+  minifyJs: false,
```

Comment:
Will the production build not have minified js and css then? That will make load times much longer

Reply:
Good catch, I changed this to false just for debugging reasons and forgot to roll it back.